week03_model_free/README.md

```diff
@@ -2,8 +2,9 @@
 ### Materials
 * Russian materials:
    - Lecture - [video](https://yadi.sk/i/jcQ1Bg8n3SrhuQ)
-   - Q-learning seminar - [video](https://yadi.sk/i/dQmolwOy3EtGNK)
-   - Sarsa & stuff - [seminar2](https://yadi.sk/i/XbqNQmjm3ExNsq)
+   - Seminars
+      - Q-learning seminar - [video](https://yadi.sk/i/dQmolwOy3EtGNK) (older track - [assignment](https://github.com/neer201/Practical_RL/tree/spring20_repair_week03/week03_model_free/crawler_and_pacman/seminar_py2))
```

Comment:
The `spring20_repair_week03` will likely be deleted after this PR is merged. Please use `spring20`, or, better yet, a specific commit hash.